tests/infra_test.go

```diff
@@ -984,3 +1037,19 @@ func validatedHTTPResponses(errorsChan chan error, concurrency int) error {
 
 	return sampledError
 }
+
+func getSupportedIP(virtClient kubecli.KubevirtClient, ips []string, family k8sv1.IPFamily) string {
+	ip := libnet.GetIp(ips, family)
+	if ip == "" {
+		if family == k8sv1.IPv4Protocol {
+			if tests.IsRunningOnKindInfraIPv6() {
```

Comment:
We should remove this method and all it usages. We don't support this cluster any more. Please don't use it.